manager/main.py

```diff
@@ -22,16 +22,18 @@
 LOGGER = get_logger(__name__)
 
 
-def create_app(spec_file):
+def create_app(spec_files):
```

Comment:
This function is imported from admin.py (manager-admin service), please change it there as well.